internal/blockprocessor/validator.go

```diff
@@ -182,6 +190,40 @@ func (v *validator) validateBlock(block *types.Block) ([]*types.ValidationInfo,
 	}
 }
 
+func (v *validator) parallelSigValidation(dataTxEnvs []*types.DataTxEnvelope) ([]*types.ValidationInfo, [][]string, error) {
+	valInfoArray := make([]*types.ValidationInfo, len(dataTxEnvs))
+	usersWithValidSigPerTX := make([][]string, len(dataTxEnvs))
+	errorArray := make([]error, len(dataTxEnvs))
```

Comment:
Would be good to avoid the type name in the variable.

```suggestion
	txsValInfo := make([]*types.ValidationInfo, len(dataTxEnvs))
	usersWithValidSigPerTX := make([][]string, len(dataTxEnvs))
	txsValError := make([]error, len(dataTxEnvs))
```

Reply:
After `usersWithValidSigPerTX` I changed it to `valInfoPerTx` & `errorPerTx`; now it's all nice and tidy.